excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/parser/ListFieldTypeParserTest.java

```diff
@@ -53,31 +55,25 @@ public void setup() {
         definitionSheet = new DefinitionSheet();
     }
 
-    @Test(expected = InvalidImportException.class)
+    @Test
     public void shouldFail_whenNoBaseTypeFoundForFixedList() {
         given(parseContext.getBaseType(BASE_FIXED_LIST)).willReturn(Optional.empty());
-
-        try {
-            new ListFieldTypeParser(parseContext, spreadsheetValidator);
-        } catch (InvalidImportException ex) {
-            Assertions.assertThat(ex).hasMessage("No base type found for: FixedList");
-            throw ex;
-        }
+        InvalidImportException ex = assertThrows(InvalidImportException.class, () ->
+            new ListFieldTypeParser(parseContext, spreadsheetValidator));
+        
+        Assertions.assertThat(ex).hasMessage("No base type found for: FixedList");
```

Comment:
Two different versions of `assertThat()` are used in this class (`org.hamcrest.MatcherAssert.assertThat` and `org.assertj.core.api.Assertions.assertThat`). Can this be simplified to use just one? The assertj one is the least frequently used one.

Reply:
Class simplified to use one version of assertThat().